HOWTO.rst

```diff
@@ -111,3 +111,57 @@ Different flavours of the documentation can be generated with wildcard entries:
     make sphinx.html # Builds HTML doc at docs/build/html/index.html
     make sphinx.epub # Builds EPUB doc at docs/build/epub
 
+----------
+Windows Troublshoot
+----------
+1. Download 7-zip from https://www.7-zip.org/download.html (to extract .zip/.tgz/.tar)
+2. Download Apache Spark from http://spark.apache.org/downloads.html (Spark Release 3.1.2, Pre-built for Hadoop 3.2 and later),
+    - Download is a .tgz file, extract it to get a .tar file and finally extract to get "spark-3.1.2-bin-hadoop3.2" folder
+    - Store this "spark-3.1.2-bin-hadoop3.2" folder in a directory like "C:\Spark\spark-3.1.2-bin-hadoop3.2"
```

Comment:
Storing in `C:\xx` is typically not a safe practice and you might be missing permissions to write to that location too.

Consider writing
```suggestion
    - Store this "spark-xxx-bin-hadoopx.x" folder in a permanent directory (example "C:\Users\xx\Spark\spark-3.1.2-bin-hadoop3.2"
```